pages/documentation/simulation/powerflow/dynaflow.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+---
+
+# DynaFlow
+
+<span style="color: red">TODO: explain what is DynaFlow, add a link to DynaFlow website</span>
+
+* TOC
+{:toc}
+
+## Installation
+
+<span style="color: red">TODO: explain how to install DynaFlow</span>
+
+## Configuration
+To use DynaFlow for all power flow computations, you have to configure the `load-flow` module in your configuration file:
+```yaml
+load-flow:
+  default-impl-name: "DynaFlow"
+```
+
+### Specific parameters
+
+**SvcRegulationOn**  
+The `SvcRegulationOn` is an optional boolean property that defines if SVCs (Static Var Compensator) take part in the voltage regulation. The default value of this parameter is `false`.
+
+**ShuntRegulationOn**  
+The `ShuntRegulationOn` is an optional boolean property that defines if Shunts take part in the voltage regulation. 
+The default value of this parameter is `false`.
+
+**AutomaticSlackBusOn**  
+The `AutomaticSlackbusOn` is an optional boolean property that defines if DynaFlow computes the slack bus (phase reference bus) by itself or if the slack bus is provided.
+The default value of this parameter is `true`.
+
+**VscAsGenerators**
+The `VscAsGenerators` is an optional boolean property that defines if VSCs (Voltage Source Converters) are modeled as generators.
+The default value of this parameter is `true`.
+
+**LccAsLoads**  
+The `LccAsLoads` is an optional boolean property that defines if LCCs (Line Commutated Converters) are modeled as loads.
+The default value of this parameter is `true`.
+
+### Generic parameters
+Furthermore, DynaFlow only supports two of the generic parameters: 
+- NoGeneratorReactiveLimits
```

Comment:
noGeneratorReactiveLimits

Reply:
If you don't want to have capital for these parameters then I think the specifics parameters ought to have their first capital removed too

Reply:
I agree, every parameter should be written with no capital letter as it is the syntax used in the yaml and json